packages/web/components/TechnologyForm/Review/helpers.js

```diff
@@ -0,0 +1,49 @@
+export const fundingData = {
+	types: [
+		{
+			value: 'public',
+			label: 'Público',
+		},
+		{
+			value: 'private',
+			label: 'Privado',
+		},
+		{
+			value: 'collective',
+			label: 'Coletivo',
+		},
+	],
+	status: [
+		{
+			value: 'not_acquired',
+			label: 'Não adquirido',
+		},
+		{
+			value: 'acquiring',
+			label: 'Em aquisição',
+		},
+		{
+			value: 'acquired',
+			label: 'Já adquirido',
+		},
+	],
+};
+
+/**
+ * Returns funding label based on value.
+ *
+ * @param {string} scope The scope of funding.
+ * @param {string} value The funding value
+ * @returns {string} The funding label.
+ */
+export const getFundingLabelByValue = (scope, value) => {
+	const keys = Object.keys(fundingData);
+
+	if (!scope || !keys.some((key) => key === scope)) {
```

Comment:
```suggestion
	if (!scope || !keys.includes(scope)) {
```

Acho que nesse caso como não temos uma busca baseado em terminada propriedade do object `key.value` por exemplo, utilizar o `.some` é meio que "overengineering" acredito eu, nesse caso o resultado vai ser o mesmo, apenas uma observação

Reply:
Concordo, na verdade nem olhei pra esse método, só peguei do Form/Review e coloquei no helper

Reply:
É, essa culpa é minha :laughing: